MVP/Repos/ScoreRepo.swift

```diff
@@ -10,71 +10,105 @@ import Foundation
 
 class ScoreRepo
```

Comment:
Should this class be getting score, leaderboard, and players? or break each call up into their own repo?